### PR TITLE
Ver 1.0.1

### PR DESCRIPTION
**Pull request by boefraty74**

### CODE REVIEW

File: script.r [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
 #DEBUG in RStudio
-# fileRda = "C:/Users/boefraty/projects/PBI/R/tempData.Rda"
-# if(file.exists(dirname(fileRda)))
-# {
-#   if(Sys.getenv("RSTUDIO")!="")
-#     load(file= fileRda)
-#   else
-#     save(list = ls(all.names = TRUE), file=fileRda)
-# }
+fileRda = "C:/Users/boefraty/projects/PBI/R/tempData.Rda"
```
Comment (ignatvilesov) [inline script.r]: It seems this code is not used. Can we remove it?

File: script.r [hunk 2]
```diff
@@ -375,9 +462,35 @@ if(validToPlot)
...
+  # lower2 = c(padNA1,lower2)
+  # upper1 = c(padNA1,upper1)
+  # upper2 = c(padNA1,upper2)
+  # 
+  # exportDF = data.frame(Date = as.character(c(x_full,f_full)),Value = c(y1,y2),
+  #                       lower1 = lower1,
+  #                       lower2 = lower2,
+  #                       upper1 = upper1,
+  #                       upper2 = upper2)
+  # colnames(exportDF)[c(1,2)] = c(labTime,labValue)
```
Comment (ignatvilesov) [inline script.r]: Can we removed commented code?

File: script.r (hunk 1 above)
Comment (boefraty74) [reply script.r]: done

File: script.r (hunk 2 above)
Comment (boefraty74) [reply script.r]: done

**Pull request merged**